enterprise/models.py

```diff
@@ -121,6 +121,13 @@ class Meta:
         )
     )
 
+    hide_audit_track = models.BooleanField(
```

Comment:
please add this new field to the [EnterpriseCustomer serializer](https://github.com/edx/edx-enterprise/blob/master/enterprise/api/v1/serializers.py#L65)